pkg/logging/logging.go

```diff
@@ -17,83 +17,55 @@
 package logging
 
 import (
-	"context"
 	"errors"
 	"fmt"
-	"os"
-	"path/filepath"
-	"strconv"
 
 	"github.com/containerd/containerd/runtime/v2/logging"
-	"github.com/containerd/nerdctl/pkg/logging/jsonfile"
-	units "github.com/docker/go-units"
-	"github.com/natefinch/lumberjack"
 )
 
 const (
 	// MagicArgv1 is the magic argv1 for the containerd runtime v2 logging plugin mode.
 	MagicArgv1 = "_NERDCTL_INTERNAL_LOGGING"
 	MaxSize    = "max-size"
 	MaxFile    = "max-file"
+	DriverType = "logging-driver"
 )
 
+const (
+	jsonFile = "json-file"
+)
+
+type NerdCTLLoggingInterface interface {
+	init(argsMap map[string]string) error
+	getLoggerFunc(argsMap map[string]string) (logging.LoggerFunc, error)
+}
+
 // Main is the entrypoint for the containerd runtime v2 logging plugin mode.
 //
 // Should be called only if argv1 == MagicArgv1.
 func Main(argsMap map[string]string) error {
-	fn, err := getLoggerFunc(argsMap)
+	if _, ok := argsMap[DriverType]; !ok {
+		return errors.New("the driverType is empty")
```

Comment:
no need for this;  driver in never, it has a default value `json-file`

Reply:
OK